test/suite/test_timestamp08.py

```diff
@@ -0,0 +1,261 @@
+        ('V2', dict(conn_config='create,log=(enabled)', using_log=True)),
+    ]
+
+    scenarios = make_scenarios(conncfg, types, ckpt)
+
+    # Check that a cursor (optionally started in a new transaction), sees the
+    # expected values.
+    def check(self, session, txn_config, tablename, expected, prn=False):
+
+        if txn_config:
+            session.begin_transaction(txn_config)
+
+        cur = session.open_cursor(tablename, None)
+        actual = dict((k, v) for k, v in cur if v != 0)
+        if prn == True:
+            print "CHECK : Expected"
+            print expected
+            print "CHECK : Actual"
+            print actual
+        self.assertTrue(actual == expected)
+        # Search for the expected items as well as iterating
+        for k, v in expected.iteritems():
+            self.assertEqual(cur[k], v, "for key " + str(k))
+        cur.close()
+        if txn_config:
+            session.commit_transaction()
+    #
+    # Take a backup of the database and verify that the value we want to
+    # check exists in the tables the expected number of times.
+    #
+    def backup_check(self, check_value, expected_ts_log, expected_ts_nolog):
+
+        newdir = "BACKUP"
+        copy_wiredtiger_home('.', newdir, True)
+
+        conn = self.setUpConnectionOpen(newdir)
+        session = self.setUpSessionOpen(conn)
+        cur_ts_log      = session.open_cursor(self.table_ts_log)
+        cur_ts_nolog    = session.open_cursor(self.table_ts_nolog)
+        # Count how many times the check_value is present in the
+        # logged timestamp table.
+        actual_ts_log = 0
+        for k, v in cur_ts_log:
+            if check_value == v:
+                actual_ts_log += 1
+        cur_ts_log.close()
+        # Count how many times the check_value is present in the
+        # not logged timestamp table
+        actual_ts_nolog = 0
+        for k, v in cur_ts_nolog:
+            if check_value == v:
+                actual_ts_nolog += 1
+        cur_ts_nolog.close()
+        conn.close()
+        self.assertEqual(actual_ts_log, expected_ts_log)
+        self.assertEqual(actual_ts_nolog, expected_ts_nolog)
+
+    # Check that a cursor sees the expected values after a checkpoint.
+    def ckpt_backup(self, check_value, valcnt_ts_log, valcnt_ts_nolog):
+
+        # Take a checkpoint.  Make a copy of the database.  Open the
+        # copy and verify whether or not the expected data is in there.
+        self.pr("CKPT: " + self.ckptcfg)
+        ckptcfg = self.ckptcfg
+
+        self.session.checkpoint(ckptcfg)
+        self.backup_check(check_value, valcnt_ts_log, valcnt_ts_nolog)
+
+    def test_timestamp08(self):
+        if not wiredtiger.timestamp_build():
+            self.skipTest('requires a timestamp build')
+
+        config_default = 'key_format=i,value_format=i,memory_page_max=32k,leaf_page_max=8k,internal_page_max=8k'
```

Comment:
config_default and config_nolog do not seem to be used anywhere. Are these base class variables?

Reply:
unused variables, removed them.